liota/lib/transports/mqtt.py

```diff
@@ -78,6 +79,14 @@ def on_disconnect(self, client, userdata, rc):
         self._connect_result_code = sys.maxsize
         self._disconnect_result_code = rc
         log.info("Disconnected with result code : {0} : {1} ".format(str(rc), paho.connack_string(rc)))
+        if self._disconnect_result_code == 0:
+            log.info("Clean disconnection")
+        else:
+            # Added a random sleep so that every client tries out reconnection to broker at different
+            # time to reduce load
+            wait_time = random.randint(1, 5)
```

Comment:
Why do we have two random functions, one with a wait_time here and another at line 283? Can't we be random enough just with one single function? Or at least have them in the same place?